effekt/jvm/src/main/scala/effekt/EffektConfig.scala

```diff
@@ -121,6 +121,10 @@ class EffektConfig(args: Seq[String]) extends REPLConfig(args) {
 
   def interpret(): Boolean = !server() && !compile() && !build()
 
+  def repl(): Boolean = filenames().isEmpty && !server() && !compile()
+
+  def exitOnError(): Boolean = !repl() && !server()
```

Comment:
Do we have a semantic name for this? What characterizes these options? Being in batch mode?